index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <link rel="stylesheet" type="text/css" href="./css/style.css">
+   <meta charset="UTF-8">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <title>Portfolio Web Page </title>
+</head>
+<body>
+   <nav id="navbar">
+      <a href="#welcome-section">Contact</a>
+      <a href="#projects">Work</a>
+      <a href="#profile-link">About</a>
+   </nav>
+   <main>
+      <section id="welcome-section">
+         <h1>Hey I am Néstor Curcho,</h1>

```

Comment:
kindly use hx tag like(h2,h3 etc..) for semantic reason.

Reply:
i used h1 in my body

any other change?